### PR TITLE
C2C-38 Cleaning and adding drugs concepts

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/conceptclasses/orderableclasses.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Uuid,Void/Retire,Name,Description
+9dc243d8-9b36-48b7-99e3-211c1b193692,,Product,Concept class for Product orders
```
Comment (rbuisson) [inline configuration/conceptclasses/orderableclasses.csv:2]: Should this file be part of the PR?

File: configuration/concepts/commons.csv [hunk 2]
```diff
@@ -1,22 +1,57 @@
...
+f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,Commentaires,See comments,Commentaires,,,Frequency,N/A,,,,,,,,,,,
+9d5b708e-3f10-11e4-adec-0800271c1b75,,CIEL:162384,Dosing Units,Unités de dosage,dosing units,Unités de dosage,,,ConvSet,N/A,,,Capsule(s); Tablet(s); ml; mg; IU; Drop(s); Tablespoon; Teaspoon; Unit(s); Vial(s); Ampule(s); Puff(s); Application; Packet; Dose; See comments,,,,,,,,
+0de96a2d-5c2c-4110-9770-914369de1713,,CIEL:162390,Opthalmic,Ophtalmique,Opthalmic,Ophtalmique,,,Misc,N/A,,,,,,,,,,,
+99cbeb1b-621c-4645-a600-d702364d9885,,CIEL:162387,Otic,Otique,Otic,Otique,,,Misc,N/A,,,,,,,,,,,
+9d6c9153-3f10-11e4-adec-0800271c1b75,,CIEL:162392,Intravaginal,Intravaginale,Intravaginal,Intravaginale,,,Misc,N/A,,,,,,,,,,,
+c5dee13f-524c-4dcc-9ad5-478a77596096,,CIEL:160240,Oral,Oral,Oral,Oral,,,Misc,N/A,,,,,,,,,,,
+1c0e27f8-63af-4bba-9629-fae799948a7c,,,Topical,Topical,Topical,Topical,,,Misc,N/A,,,,,,,,,,,
+09ddefb3-5f66-48f3-92c6-44f8391b9b97,,CIEL:160241,Inhalation,Inhalation,Inhalation,Inhalation,,,Misc,N/A,,,,,,,,,,,
+2bd7f0b3-57d1-421c-aac6-762ddd77cf4b,,CIEL:160243,Intramuscular,Intramuscular,Intramuscular,Intramuscular,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Intramusculaire", short name would be "IM"

File: configuration/concepts/commons.csv [hunk 3]
```diff
@@ -1,22 +1,57 @@
...
+76bb4178-99cd-4999-a8c7-bf982ba4b7b0,,CIEL:160242,Intravenous,Intravenous,Intravenous,Intravenous,,,Misc,N/A,,,,,,,,,,,
+7f5650cc-0549-4839-bb00-7aa2503bcedf,,CIEL:160245,Subcutaneous,Subcutaneous,Subcutaneous,Subcutaneous,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Sous-cutanée" short name "S/C"

File: configuration/concepts/commons.csv [hunk 4]
```diff
@@ -1,22 +1,57 @@
...
+9d5b708e-3f10-11e4-adec-0800271c1b75,,CIEL:162384,Dosing Units,Unités de dosage,dosing units,Unités de dosage,,,ConvSet,N/A,,,Capsule(s); Tablet(s); ml; mg; IU; Drop(s); Tablespoon; Teaspoon; Unit(s); Vial(s); Ampule(s); Puff(s); Application; Packet; Dose; See comments,,,,,,,,
+0de96a2d-5c2c-4110-9770-914369de1713,,CIEL:162390,Opthalmic,Ophtalmique,Opthalmic,Ophtalmique,,,Misc,N/A,,,,,,,,,,,
+99cbeb1b-621c-4645-a600-d702364d9885,,CIEL:162387,Otic,Otique,Otic,Otique,,,Misc,N/A,,,,,,,,,,,
+9d6c9153-3f10-11e4-adec-0800271c1b75,,CIEL:162392,Intravaginal,Intravaginale,Intravaginal,Intravaginale,,,Misc,N/A,,,,,,,,,,,
+c5dee13f-524c-4dcc-9ad5-478a77596096,,CIEL:160240,Oral,Oral,Oral,Oral,,,Misc,N/A,,,,,,,,,,,
+1c0e27f8-63af-4bba-9629-fae799948a7c,,,Topical,Topical,Topical,Topical,,,Misc,N/A,,,,,,,,,,,
+09ddefb3-5f66-48f3-92c6-44f8391b9b97,,CIEL:160241,Inhalation,Inhalation,Inhalation,Inhalation,,,Misc,N/A,,,,,,,,,,,
+2bd7f0b3-57d1-421c-aac6-762ddd77cf4b,,CIEL:160243,Intramuscular,Intramuscular,Intramuscular,Intramuscular,,,Misc,N/A,,,,,,,,,,,
+76bb4178-99cd-4999-a8c7-bf982ba4b7b0,,CIEL:160242,Intravenous,Intravenous,Intravenous,Intravenous,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Intraveineuse" short name "IV"

File: configuration/concepts/commons.csv [hunk 5]
```diff
@@ -1,22 +1,57 @@
...
+7f5650cc-0549-4839-bb00-7aa2503bcedf,,CIEL:160245,Subcutaneous,Subcutaneous,Subcutaneous,Subcutaneous,,,Misc,N/A,,,,,,,,,,,
+9d697972-3f10-11e4-adec-0800271c1b75,,CIEL:162394,Drug Routes,Voie d'administration,Drug Routes,Voie d'administration,,,Misc,N/A,,,Oral; Topical; Inhalation; Intramuscular; Intravenous; Intravaginal; Opthalmic; Otic; Subcutaneous; See comments;,,,,,,,,
+c3cfe994-bdb2-4024-9b54-3f7cebd465ca,,,Before meals,Avant les repas,ac,ac,,,Misc,N/A,,,,,,,,,,,
+9c60a0bf-e9eb-49bd-83cc-a07b46638a72,,,Empty stomach,L'estomac vide,Empty stomach,L'estomac vide,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Estomac vide"

File: configuration/concepts/commons.csv [hunk 6]
```diff
@@ -1,22 +1,57 @@
...
+9d6c9153-3f10-11e4-adec-0800271c1b75,,CIEL:162392,Intravaginal,Intravaginale,Intravaginal,Intravaginale,,,Misc,N/A,,,,,,,,,,,
+c5dee13f-524c-4dcc-9ad5-478a77596096,,CIEL:160240,Oral,Oral,Oral,Oral,,,Misc,N/A,,,,,,,,,,,
+1c0e27f8-63af-4bba-9629-fae799948a7c,,,Topical,Topical,Topical,Topical,,,Misc,N/A,,,,,,,,,,,
+09ddefb3-5f66-48f3-92c6-44f8391b9b97,,CIEL:160241,Inhalation,Inhalation,Inhalation,Inhalation,,,Misc,N/A,,,,,,,,,,,
+2bd7f0b3-57d1-421c-aac6-762ddd77cf4b,,CIEL:160243,Intramuscular,Intramuscular,Intramuscular,Intramuscular,,,Misc,N/A,,,,,,,,,,,
+76bb4178-99cd-4999-a8c7-bf982ba4b7b0,,CIEL:160242,Intravenous,Intravenous,Intravenous,Intravenous,,,Misc,N/A,,,,,,,,,,,
+7f5650cc-0549-4839-bb00-7aa2503bcedf,,CIEL:160245,Subcutaneous,Subcutaneous,Subcutaneous,Subcutaneous,,,Misc,N/A,,,,,,,,,,,
+9d697972-3f10-11e4-adec-0800271c1b75,,CIEL:162394,Drug Routes,Voie d'administration,Drug Routes,Voie d'administration,,,Misc,N/A,,,Oral; Topical; Inhalation; Intramuscular; Intravenous; Intravaginal; Opthalmic; Otic; Subcutaneous; See comments;,,,,,,,,
+c3cfe994-bdb2-4024-9b54-3f7cebd465ca,,,Before meals,Avant les repas,ac,ac,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Avant le repas"

File: configuration/concepts/commons.csv [hunk 7]
```diff
@@ -1,22 +1,57 @@
...
+9c60a0bf-e9eb-49bd-83cc-a07b46638a72,,,Empty stomach,L'estomac vide,Empty stomach,L'estomac vide,,,Misc,N/A,,,,,,,,,,,
+1351b170-9d2c-4b43-a10d-f3755a2aa936,,,After meals,Après les repas,pc,pc,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Après le repas"

File: configuration/concepts/commons.csv [hunk 8]
```diff
@@ -1,22 +1,57 @@
...
-9d8e6918-3f10-11e4-adec-0800271c1b75,,CIEL:161235,Now,Maintenant,STAT,STAT,,,Frequency,N/A,,,,,,,,,,,
 2a148983-c143-4649-b4be-f8845c934ec0,,,Nightly,Chaque nuit,Nightly,Chaque nuit,,,Frequency,N/A,,,,,,,,,,,
+f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,Commentaires,See comments,Commentaires,,,Frequency,N/A,,,,,,,,,,,
+9d5b708e-3f10-11e4-adec-0800271c1b75,,CIEL:162384,Dosing Units,Unités de dosage,dosing units,Unités de dosage,,,ConvSet,N/A,,,Capsule(s); Tablet(s); ml; mg; IU; Drop(s); Tablespoon; Teaspoon; Unit(s); Vial(s); Ampule(s); Puff(s); Application; Packet; Dose; See comments,,,,,,,,
+0de96a2d-5c2c-4110-9770-914369de1713,,CIEL:162390,Opthalmic,Ophtalmique,Opthalmic,Ophtalmique,,,Misc,N/A,,,,,,,,,,,
+99cbeb1b-621c-4645-a600-d702364d9885,,CIEL:162387,Otic,Otique,Otic,Otique,,,Misc,N/A,,,,,,,,,,,
+9d6c9153-3f10-11e4-adec-0800271c1b75,,CIEL:162392,Intravaginal,Intravaginale,Intravaginal,Intravaginale,,,Misc,N/A,,,,,,,,,,,
+c5dee13f-524c-4dcc-9ad5-478a77596096,,CIEL:160240,Oral,Oral,Oral,Oral,,,Misc,N/A,,,,,,,,,,,
+1c0e27f8-63af-4bba-9629-fae799948a7c,,,Topical,Topical,Topical,Topical,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: This one I am not sure, I'll ask around and let you know.

File: configuration/concepts/drugs_concepts.csv [hunk 9]
```diff
@@ -286,3 +283,27 @@ e28a653a-d7fc-4158-989f-9515963ea3db,,,Triple antibiotic,Antibiotique triple,Dru
 70f0f39c-9a43-4902-9f56-d7f8007f5c6c,,,Eye glasses,Lunettes,Drug,N/A,,
 3fc18bdb-948e-4573-add5-37d75853daf6,,,Orthopedic device,Appareil orthopédique,Drug,N/A,,
 b88ca045-2707-4fff-b383-89840293334d,,,Heatwrap,Heatwrap,Drug,N/A,,
+84bb5c70-878b-45c3-96e7-3282e4121e02,,,Alpha Methyl Dopa,Alpha Métyl Dopa,Drug,N/A,,
+07470e7f-f959-4344-89f6-8e6a0f442bef,,,Antipyrine,Antipyrine,Drug,N/A,,
+66de98f3-03e1-4733-9534-1f59dee19429,,,Apetamine,Apetamine,Drug,N/A,,
+a6ba4400-1db2-4d47-854f-3342204995ba,,,Apressoline,Apressoline,Drug,N/A,,
+ca2598c9-64d9-475a-b506-b6cc81741a61,,,Bebe Sirop,Bebe Sirop,Drug,N/A,,
```
Comment (rbuisson) [inline configuration/concepts/drugs_concepts.csv]: "Bébé Sirop"

**Review by rbuisson — CHANGES_REQUESTED** [review]
Made some remarks in fr.
Thanks.

File: configuration/conceptclasses/orderableclasses.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/conceptclasses/orderableclasses.csv:2]: Some of the products were submitted via the drugs...this is why i added the product related items in this PR too

File: configuration/concepts/commons.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 7 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/drugs_concepts.csv (hunk 9 above)
Comment (reagan-meant) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/commons.csv (hunk 8 above)
Comment (rbuisson) [reply configuration/concepts/commons.csv]: ok, in fr that'd be "topique"

File: configuration/concepts/commons.csv [hunk 10]
```diff
@@ -1,22 +1,57 @@
...
 335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+90233a68-c4c1-48fb-a210-2cb0d5544e00,,,Five times/Day,Five times/Day,Five times/Day,Five times/Day,,,Frequency,N/A,,,,,,,,,,,
+863264e3-1b95-441a-997d-7fb0a240613c,,,Q1H,Q1H,Q1H,Q1H,,,Frequency,N/A,,,,,,,,,,,
+d4dc0576-7a0d-43da-a123-19b26119cc6a,,,Q2H,Q2H,Q2H,Q2H,,,Frequency,N/A,,,,,,,,,,,
+4adb2e4e-40c8-4889-81ac-99b940897f54,,,Q3H,Q3H,Q3H,Q3H,,,Frequency,N/A,,,,,,,,,,,
+2c4b7e6b-1bdd-4fe9-abf0-ced777b6caf9,,,Q5H,Q5H,Q5H,Q5H,,,Frequency,N/A,,,,,,,,,,,
+8116d2c6-a183-4132-900d-02cc54a1821a,,,Monthly,Monthly,Monthly,Monthly,,,Frequency,N/A,,,,,,,,,,,
+b22ec437-b389-4875-86f9-9662732e384d,,,Twice/Week,Twice/Week,Twice/Week,Twice/Week,,,Frequency,N/A,,,,,,,,,,,
+3a427f19-979b-43f4-8e91-0fbdcf4a3773,,,Thrice/Week,Thrice/Week,Thrice/Week,Thrice/Week,,,Frequency,N/A,,,,,,,,,,,
+9261ebac-541a-4a96-b245-07b9adcccb99,,,Once Every 2 Weeks,Once Every 2 Weeks,Once Every 2 Weeks,Once Every 2 Weeks,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: in fr "Une fois toute les 2 semaines"

File: configuration/concepts/commons.csv [hunk 11]
```diff
@@ -1,22 +1,57 @@
...
+023570b0-872f-414d-88f1-6050b39e79f3,,,Once Every 3 Weeks,Once Every 3 Weeks,Once Every 3 Weeks,Once Every 3 Weeks,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: in fr "Une fois toute les 3 semaines"

File: configuration/concepts/commons.csv [hunk 12]
```diff
@@ -1,22 +1,57 @@
...
+334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,BD,Q12H,BD,,,Frequency,N/A,,,,,,,,,,,
 335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+90233a68-c4c1-48fb-a210-2cb0d5544e00,,,Five times/Day,Five times/Day,Five times/Day,Five times/Day,,,Frequency,N/A,,,,,,,,,,,
+863264e3-1b95-441a-997d-7fb0a240613c,,,Q1H,Q1H,Q1H,Q1H,,,Frequency,N/A,,,,,,,,,,,
+d4dc0576-7a0d-43da-a123-19b26119cc6a,,,Q2H,Q2H,Q2H,Q2H,,,Frequency,N/A,,,,,,,,,,,
+4adb2e4e-40c8-4889-81ac-99b940897f54,,,Q3H,Q3H,Q3H,Q3H,,,Frequency,N/A,,,,,,,,,,,
+2c4b7e6b-1bdd-4fe9-abf0-ced777b6caf9,,,Q5H,Q5H,Q5H,Q5H,,,Frequency,N/A,,,,,,,,,,,
+8116d2c6-a183-4132-900d-02cc54a1821a,,,Monthly,Monthly,Monthly,Monthly,,,Frequency,N/A,,,,,,,,,,,
+b22ec437-b389-4875-86f9-9662732e384d,,,Twice/Week,Twice/Week,Twice/Week,Twice/Week,,,Frequency,N/A,,,,,,,,,,,
+3a427f19-979b-43f4-8e91-0fbdcf4a3773,,,Thrice/Week,Thrice/Week,Thrice/Week,Thrice/Week,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "3 fois par semaine" or "3 fois/semaine"

File: configuration/concepts/commons.csv [hunk 13]
```diff
@@ -1,22 +1,57 @@
...
+334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,TDS,Q8H,TDS,,,Frequency,N/A,,,,,,,,,,,
+334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,BD,Q12H,BD,,,Frequency,N/A,,,,,,,,,,,
 335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+90233a68-c4c1-48fb-a210-2cb0d5544e00,,,Five times/Day,Five times/Day,Five times/Day,Five times/Day,,,Frequency,N/A,,,,,,,,,,,
+863264e3-1b95-441a-997d-7fb0a240613c,,,Q1H,Q1H,Q1H,Q1H,,,Frequency,N/A,,,,,,,,,,,
+d4dc0576-7a0d-43da-a123-19b26119cc6a,,,Q2H,Q2H,Q2H,Q2H,,,Frequency,N/A,,,,,,,,,,,
+4adb2e4e-40c8-4889-81ac-99b940897f54,,,Q3H,Q3H,Q3H,Q3H,,,Frequency,N/A,,,,,,,,,,,
+2c4b7e6b-1bdd-4fe9-abf0-ced777b6caf9,,,Q5H,Q5H,Q5H,Q5H,,,Frequency,N/A,,,,,,,,,,,
+8116d2c6-a183-4132-900d-02cc54a1821a,,,Monthly,Monthly,Monthly,Monthly,,,Frequency,N/A,,,,,,,,,,,
+b22ec437-b389-4875-86f9-9662732e384d,,,Twice/Week,Twice/Week,Twice/Week,Twice/Week,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "2 fois par semaine" or "2 fois/semaine"

File: configuration/concepts/commons.csv [hunk 14]
```diff
@@ -1,22 +1,57 @@
...
+334dbd41-8a92-11e4-977f-0800271c1b75,,CIEL:160870,Q6H,QID,Q6H,QID,,,Frequency,N/A,,,,,,,,,,,
+334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,TDS,Q8H,TDS,,,Frequency,N/A,,,,,,,,,,,
+334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,BD,Q12H,BD,,,Frequency,N/A,,,,,,,,,,,
 335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+90233a68-c4c1-48fb-a210-2cb0d5544e00,,,Five times/Day,Five times/Day,Five times/Day,Five times/Day,,,Frequency,N/A,,,,,,,,,,,
+863264e3-1b95-441a-997d-7fb0a240613c,,,Q1H,Q1H,Q1H,Q1H,,,Frequency,N/A,,,,,,,,,,,
+d4dc0576-7a0d-43da-a123-19b26119cc6a,,,Q2H,Q2H,Q2H,Q2H,,,Frequency,N/A,,,,,,,,,,,
+4adb2e4e-40c8-4889-81ac-99b940897f54,,,Q3H,Q3H,Q3H,Q3H,,,Frequency,N/A,,,,,,,,,,,
+2c4b7e6b-1bdd-4fe9-abf0-ced777b6caf9,,,Q5H,Q5H,Q5H,Q5H,,,Frequency,N/A,,,,,,,,,,,
+8116d2c6-a183-4132-900d-02cc54a1821a,,,Monthly,Monthly,Monthly,Monthly,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Mensuellement

File: configuration/concepts/commons.csv [hunk 15]
```diff
@@ -1,22 +1,57 @@
...
 9d77abd7-3f10-11e4-adec-0800271c1b75,,CIEL:160862,Daily,Quotidien,OD,OD,,,Frequency,N/A,,,,,,,,,,,
 334ce904-8a92-11e4-977f-0800271c1b75,,,Q4H,Q4H,Q4H,Q4H,,,Frequency,N/A,,,,,,,,,,,
-334dbd41-8a92-11e4-977f-0800271c1b75,,CIEL:160870,Q6H,Q6H,Q6H,Q6H,,,Frequency,N/A,,,,,,,,,,,
-334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,Q8H,Q8H,Q8H,,,Frequency,N/A,,,,,,,,,,,
-334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,Q12H,Q12H,Q12H,,,Frequency,N/A,,,,,,,,,,,
+334dbd41-8a92-11e4-977f-0800271c1b75,,CIEL:160870,Q6H,QID,Q6H,QID,,,Frequency,N/A,,,,,,,,,,,
+334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,TDS,Q8H,TDS,,,Frequency,N/A,,,,,,,,,,,
+334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,BD,Q12H,BD,,,Frequency,N/A,,,,,,,,,,,
 335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+90233a68-c4c1-48fb-a210-2cb0d5544e00,,,Five times/Day,Five times/Day,Five times/Day,Five times/Day,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Cinq fois/jour" or "5 fois/jour"

File: configuration/concepts/commons.csv [hunk 16]
```diff
@@ -1,22 +1,57 @@
 Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:500
+c2ba3844-98b7-4d3c-b98f-4adfc46658ee,,CIEL:1608,Capsule(s),Capsule(s),Capsule(s),Capsule(s),,,Misc,N/A,,,,,,,,,,,
+6fbbb779-94ca-4d83-8e02-8a559627b9fb,,CIEL:1513,Tablet(s),Tablet(s),Tablet(s),Tablet(s),,,Misc,N/A,,,,,,,,,,,
+22109614-7512-4739-b28a-783354d19aa1,,CIEL:162263,ml,ml,ml,ml,,,Misc,N/A,,,,,,,,,,,
+d34c85d0-f1c5-44d1-bfe3-d19d5ec875b1,,CIEL:161553,mg,mg,mg,mg,,,Misc,N/A,,,,,,,,,,,
+8574ffb1-222d-4a11-97c9-7e401c59e2b1,,CIEL:162264,IU,IU,IU,IU,,,Misc,N/A,,,,,,,,,,,
+2d735487-2f66-412c-a615-2873ddf94dfd,,CIEL:162356,Drop(s),Drop(s),Drop(s),Drop(s),,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Goutte(s)

File: configuration/concepts/commons.csv [hunk 17]
```diff
@@ -1,22 +1,57 @@
...
+5de903e6-2a66-4253-891d-86558e1282d2,,CIEL:162382,Vial(s),Vial(s),Vial(s),Vial(s),,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Fiole(s)

File: configuration/concepts/commons.csv [hunk 18]
```diff
@@ -1,22 +1,57 @@
...
+6705f17a-242e-44dc-87fa-ed6a15f2a1cb,,CIEL:162335,Ampule(s),Ampule(s),Ampule(s),Ampule(s),,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Ampoule(s)

File: configuration/concepts/commons.csv [hunk 19]
```diff
@@ -1,22 +1,57 @@
...
+6fa802c9-5ad8-4e0b-9b8b-d80e4d91b21d,,CIEL:162378,Tablespoon,Tablespoon,Tablespoon,Tablespoon,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Cuillère(s) à soupe"

File: configuration/concepts/commons.csv [hunk 20]
```diff
@@ -1,22 +1,57 @@
...
+89a91a0c-35d4-45b9-a87d-dac77df05c81,,CIEL:162379,Teaspoon,Teaspoon,Teaspoon,Teaspoon,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "Cuillère(s) à café"

File: configuration/concepts/commons.csv [hunk 21]
```diff
@@ -1,22 +1,57 @@
...
+ffe07e7c-0452-4e59-b326-9d2853b56417,,CIEL:162381,Unit(s),Unit(s),Unit(s),Unit(s),,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Unité(s)

File: configuration/concepts/commons.csv [hunk 22]
```diff
@@ -1,22 +1,57 @@
...
+5fa6126f-dc1a-42d4-bfda-b3b9e2cd3b90,,CIEL:162372,Puff(s),Puff(s),Puff(s),Puff(s),,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Bouffée(s)

**Review by rbuisson — CHANGES_REQUESTED** [review]
Looks good. Just added some translations I've missed

File: configuration/concepts/commons.csv (hunk 8 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 10 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 11 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 12 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 13 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 14 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 15 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 16 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 17 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 18 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 19 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 20 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 21 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 22 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv [hunk 23]
```diff
@@ -1,22 +1,57 @@
...
 9d77abd7-3f10-11e4-adec-0800271c1b75,,CIEL:160862,Daily,Quotidien,OD,OD,,,Frequency,N/A,,,,,,,,,,,
 334ce904-8a92-11e4-977f-0800271c1b75,,,Q4H,Q4H,Q4H,Q4H,,,Frequency,N/A,,,,,,,,,,,
-334dbd41-8a92-11e4-977f-0800271c1b75,,CIEL:160870,Q6H,Q6H,Q6H,Q6H,,,Frequency,N/A,,,,,,,,,,,
-334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,Q8H,Q8H,Q8H,,,Frequency,N/A,,,,,,,,,,,
-334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,Q12H,Q12H,Q12H,,,Frequency,N/A,,,,,,,,,,,
+334dbd41-8a92-11e4-977f-0800271c1b75,,CIEL:160870,Q6H,QID,Q6H,QID,,,Frequency,N/A,,,,,,,,,,,
+334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,TDS,Q8H,TDS,,,Frequency,N/A,,,,,,,,,,,
+334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,BD,Q12H,BD,,,Frequency,N/A,,,,,,,,,,,
 335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+90233a68-c4c1-48fb-a210-2cb0d5544e00,,,Five times/Day,Cinq fois/jour,Five times/Day,Cinq fois/jour,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: 5 times/Day
5 fois/jour

File: configuration/concepts/commons.csv [hunk 24]
```diff
@@ -1,22 +1,57 @@
...
+863264e3-1b95-441a-997d-7fb0a240613c,,,Q1H,Q1H,Q1H,Q1H,,,Frequency,N/A,,,,,,,,,,,
+d4dc0576-7a0d-43da-a123-19b26119cc6a,,,Q2H,Q2H,Q2H,Q2H,,,Frequency,N/A,,,,,,,,,,,
+4adb2e4e-40c8-4889-81ac-99b940897f54,,,Q3H,Q3H,Q3H,Q3H,,,Frequency,N/A,,,,,,,,,,,
+2c4b7e6b-1bdd-4fe9-abf0-ced777b6caf9,,,Q5H,Q5H,Q5H,Q5H,,,Frequency,N/A,,,,,,,,,,,
+8116d2c6-a183-4132-900d-02cc54a1821a,,,Monthly,Mensuellement,Monthly,Mensuellement,,,Frequency,N/A,,,,,,,,,,,
+b22ec437-b389-4875-86f9-9662732e384d,,,Twice/Week,2 fois/semaine,Twice/Week,2 fois/semaine,,,Frequency,N/A,,,,,,,,,,,
+3a427f19-979b-43f4-8e91-0fbdcf4a3773,,,Thrice/Week,3 fois/semaine,Thrice/Week,3 fois/semaine,,,Frequency,N/A,,,,,,,,,,,
+9261ebac-541a-4a96-b245-07b9adcccb99,,,Once Every 2 Weeks,Une fois toute les 2 semaines,Once Every 2 Weeks,Une fois toute les 2 semaines,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "1 fois toutes les 2 semaines"

File: configuration/concepts/commons.csv [hunk 25]
```diff
@@ -1,22 +1,57 @@
...
+023570b0-872f-414d-88f1-6050b39e79f3,,,Once Every 3 Weeks,Une fois toute les 3 semaines,Once Every 3 Weeks,Une fois toute les 3 semaines,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: "1 fois toutes les 3 semaines"

**Review by rbuisson — CHANGES_REQUESTED** [review]
Minor changes.

File: configuration/concepts/commons.csv (hunk 23 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 24 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 25 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

**Pull request merged**